markdown-code-checker.py

```diff
@@ -0,0 +1,90 @@
+import marko as md
```

Comment:
Having this script at the top level is a bit unexpected / noisy. Please put it inside a new `./scripts` folder. (We already do this in some other repos).